hedera-node/src/main/java/com/hedera/services/state/expiry/ExpiringEntity.java

```diff
@@ -0,0 +1,50 @@
+package com.hedera.services.state.expiry;
+
+import com.hedera.services.state.submerkle.EntityId;
+
+import java.util.Objects;
+import java.util.function.Consumer;
+
+public class ExpiringEntity {
+    private final Consumer<EntityId> consumer;
+    private final EntityId id;
+    private final long expiry;
+
+    public ExpiringEntity(EntityId id, Consumer<EntityId> consumer, long expiry) {
+        this.consumer = consumer;
+        this.id = id;
+        this.expiry = expiry;
+    }
+
+    public long expiry() {
+        return expiry;
+    }
+
+    public EntityId id() {
+        return id;
+    }
+
+    public Consumer<EntityId> consumer() {
+        return consumer;
+    }
+
+    @Override
```

Comment:
Do we need to override `hashCode` and `equals` for the `ExpiringEntity` type? It seems to be a process object, not a value object. 

(That is, each instance of `ExpiringEntity` will represent a different piece of information about how expiries should be processed...unlike say `String`, where `var a = "same"` and `var b = a.substring(0, 2) + "me"` are different instances, but should both represent the same **value**.)